src/webapp/src/website/controllers/authentication.controller.ts

```diff
@@ -60,13 +60,11 @@ export class AuthenticationController {
   @Post('/signup')
   async postSignup (@Request() req, @Res() res: Response): Promise<any> {
     const { email, password, account } = req.body
+    const data = await this.settingsService.getWebsiteRenderingVariables()
 
     const user = await this.authService.registerUser(email, password, account)
     if (user == null) { // TODO: redirect to error page
```

Comment:
We really must have different error values because now anything that fails is considered as `User already registered`